### DISCUSSION TITLE
Grocy Addon: how to identify the ip

### DISCUSSION THREAD

**Discussion by johnnyeightpence** [hassio-addons/addon-grocy#281] [Q&A]
Hi
may be this is a stupid question but I am lost.
I installed grocy on my Home Assistant as an Addon and would like to connect the android app to the instance too. 
I created the API Key and scanned the Barcode on my mobile. 
Strangely there is an IP address mentioned in the config window which does not appear in my network. 
I assumed that the addon would have the same IP as the Home Assistant and just the port would be different. 
When I press "connect" it tries something for a minute or two and after that I get the message that "connection time expired". 

What am I doing wrong? How can I connect the android app with my grocy installation?

**Comment by hklages** [comment]
The add on portainer is easy to install and shows you the ipv4 of all container. Home Assistant Add ons run in a separate network. HA is the gateway.

**Reply by DjordjeMandic** [reply]
does that mean you cannot connect grocy android app to grocy addon?
